src/test/java/org/ohdsi/circe/check/checkers/UnusedConceptsCheckTest.java

```diff
@@ -0,0 +1,224 @@
+package org.ohdsi.circe.check.checkers;
+
+import static org.junit.Assert.*;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Test;
+import org.ohdsi.circe.check.Checker;
+import org.ohdsi.circe.check.Warning;
+import org.ohdsi.circe.cohortdefinition.CohortExpression;
+
+public class UnusedConceptsCheckTest {
+
+
+    private static final String EXPRESSION = "{\n" +
```

Comment:
I would move it into resources and read it from there

Reply:
Agreed, any content that would be loaded as a file, just make it an external resource (like our SQL templates are stored).